src/averageAges.js

```diff
@@ -38,6 +64,34 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const ages = [];
+  let averageAge = 0;
+  let eteration = 0;
+  const mothers = people.map((human) => human.mother);
+  const isWoman = human => human.sex === 'f';
```

Comment:
* Consider creating helper functions and constants variables before your task functions and reuse them inside calculateWomenAverageAge, calculateMenAverageAge, etc.
* Consider using constants instead of magic numbers, it's an antipattern
```suggestion
  const isWoman = human => human.sex === WOMAN;
```